README.md

```diff
@@ -1,27 +1,32 @@
-GitHub raw file proxy
+Git raw file proxy
 =====================
 
-This simple application can be used to proxy private raw files from GitHub.
+This simple application can be used to proxy private raw files from A Git provider.
 
 Preparation
 -----------
 
 Use the documentation of the API to create an access token that has access with the 
 scope `repo` to the target repository.
 
-http://developer.github.com/v3/oauth/#get-a-single-authorization
+Github: http://developer.github.com/v3/oauth/#get-a-single-authorization
+Gitlab: https://docs.gitlab.com/ee/api/
 
 Configuration
 -------------
 
 ``` scala
-	allowed.accessTokens="3ab423aa747918asdd2a256b852c5a60580d3,3ab423aa7479186c96eb32asd5a60asd0d3"
+allowed.accessTokens="3ab423aa747918asdd2a256b852c5a60580d3,3ab423aa7479186c96eb32asd5a60asd0d3"
 ```
 
 Usage
 -----
 
-http://localhost:9000/accessToken/Owner/Repository/path/to/file.ext
+On the master branch
+http://localhost:9000/{accessToken}/{Owner}/{Repository}/{path/to/file.ext}
+
+On a specify branch
+http://localhost:9000/branch/{branchName}/{accessToken}/{Owner}/{Repository}/{path/to/file.ext}
```

Comment:
I also thought that would be best, but it wont work. Since what is the difference between branch/path and /path. Path can also contain the path separator.